README.md

```diff
@@ -318,14 +324,15 @@ Multiple keys can be provided, either in an Array or as extra arguments.
 ### asMutable
 
 ```javascript
-var mutableObject = Immutable({when: "the", levee: "breaks"}).asMutable();
+var obj = Immutable({when: "the", levee: "breaks"});
+var mutable = Immutable.asMutable(obj);
 
 mutableObject.have = "no place to go";
 
 mutableObject // {when: "the", levee: "breaks", have: "no place to go"}
 ```
 
-Returns a mutable copy of the object. For a deeply mutable copy, in which any instances of `Immutable` contained in nested data structures within the object have been converted back to mutable data structures, call `.asMutable({deep: true})` instead.
+Returns a mutable copy of the object. For a deeply mutable copy, in which any instances of `Immutable` contained in nested data structures within the object have been converted back to mutable data structures, call `.asMutable(obj, {deep: true})` instead.
```

Comment:
I think this was written as `.asMutable` because the object to the left of the dot could be anything; with the docs only talking about the static method syntax, this could be changed to `Immutable.asMutable`